### PR TITLE
Add method to TweaksConfigurationsCoordinator for UnitTests

### PR DESCRIPTION
**Pull request by AlexChe-je**
- Added isTweaksCacheEmpty method for UnitTest
- Bumped version

### CODE REVIEW

File: Example/Podfile.lock [hunk 1]
```diff
@@ -1,63 +1,109 @@
...
+  GoogleAppMeasurement: 5029d78f2b438bd30ffb44e6c558b57c21622727
+  GoogleUtilities: 95996bea7c7d9b8fb811b7507669a4a8762f80c7
+  JustTweak: df75851891f4e6244ad175cfa8608589c2c38ed3
+  nanopb: 2901f78ea1b7b4015c860c2fdd1ea2fee1a18d48
+  Protobuf: 1eb9700044745f00181c136ef21b8ff3ad5a0fd5
 
 PODFILE CHECKSUM: 2db0e6c19b4429f2f441eefce47a302d49d3b5d3
 
-COCOAPODS: 1.3.1
+COCOAPODS: 1.5.3
```
Comment (albertodebortoli-je) [inline Example/Podfile.lock]: @AlexChe-je please revert to Cocoapods 1.3.1 for this PR. We can update it in a separate one later.

File: Example/JustTweak.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -437,14 +389,14 @@
 			);
 			inputPaths = (
 				"${SRCROOT}/Pods/Target Support Files/Pods-JustTweak_Example/Pods-JustTweak_Example-frameworks.sh",
-				"${BUILT_PRODUCTS_DIR}/GoogleToolboxForMac/GoogleToolboxForMac.framework",
+				"${BUILT_PRODUCTS_DIR}/GoogleUtilities/GoogleUtilities.framework",
```
Comment (albertodebortoli-je) [inline Example/JustTweak.xcodeproj/project.pbxproj]: Not sure why this is part of the PR but this is alla unrelated. It shouldn't happen even with Cocoapods 1.5, I trust reverting back to 1.3.1 will fix it.

File: JustTweak.podspec [hunk 3]
```diff
@@ -1,7 +1,7 @@
 
 Pod::Spec.new do |s|
   s.name             = 'JustTweak'
-  s.version          = '2.0.0'
+  s.version          = '2.0.1'
```
Comment (albertodebortoli-je) [inline JustTweak.podspec]: On open source project we don't bump the version with each PR.

File: JustTweak/Classes/Core/TweaksConfigurationsCoordinator.swift [hunk 4]
```diff
@@ -128,4 +128,7 @@ import Foundation
         tweaksCache = [String : TweakCachedValue]()
     }
     
+    @objc public func isTweaksCacheEmpty() -> Bool {
```
Comment (albertodebortoli-je) [inline JustTweak/Classes/Core/TweaksConfigurationsCoordinator.swift]: If this method is to ease the unit tests, it should be internal. Mind that #5 also removed @objc and @objcMembers.

Comment (AlexChe-je) [reply JustTweak/Classes/Core/TweaksConfigurationsCoordinator.swift]: I am going to put all unit test in other pr - to a JustEat Tests. But I need this method  to implement Unit test in JustEat Tests.

File: Example/Podfile.lock (hunk 1 above)
Comment (albertodebortoli-je) [reply Example/Podfile.lock]: @AlexChe-je I know, but you just have to use the correct version.

**Comment by albertodebortoli-je** [comment]
Closing as discussed as not relevant.

**Pull request closed without merging**